src/compiler/scala/tools/nsc/Global.scala

```diff
@@ -1297,7 +1297,7 @@ class Global(var currentSettings: Settings, var reporter: Reporter)
 
     /** does this run compile given class, module, or case factory? */
     // NOTE: Early initialized members temporarily typechecked before the enclosing class, see typedPrimaryConstrBody!
-    //       Here we work around that wrinkle by claiming that a early-initialized member is compiled in
+    //       Here we work around that wrinkle by claiming that an early-initialized member is compiled in
```

Comment:
I would say the hyphen is correct (because early modifies initialized and not member) but the term is an "early definition" and Prog in Scala calls them "pre-initialized".